src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_with_ppg.yaml

```diff
@@ -191,10 +162,23 @@ interactions:
       x-ms-ratelimit-remaining-subscription-writes:
       - '1199'
     status:
-      code: 201
-      message: Created
+      code: 400
```

Comment:
why bad request.

Reply:
This is the response which containing the error message 
> principal clientID: http://clitest000002 not found in Active Directory tenant

This has been solved by setting principal in the environment variable